next-tavla/src/Shared/styles/spacing.css

```diff
@@ -215,13 +215,25 @@
 }
 
 .w-100 {
```

Comment:
w-120 her, siden width blir satt til 120%, eller? har ikke så mye å si for min del, men bare lurer 😅